python/nabo.cpp

```diff
@@ -55,25 +70,25 @@ MappedEigenDoubleMatrix* eigenFromBoostPython(const object cloudIn, const char *
 {
 	int dimCount, pointCount;
 	const PyObject *cloudObj(cloudIn.ptr());
-	
+
```

Comment:
This and followings are pure whitespace changes, I think they should not be in the patch.